pkgs/data/icons/phinger-cursors/default.nix

```diff
@@ -0,0 +1,27 @@
+{ lib, stdenvNoCC, fetchurl }:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "phinger-cursors";
+  version = "1.1";
+
+  src = fetchurl {
+    url = "https://github.com/phisch/phinger-cursors/releases/download/v${version}/phinger-cursors-variants.tar.bz2";
+    sha256 = "sha256-II+1x+rcjGRRVB8GYkVwkKVHNHcNaBKRb6C613901oc=";
+  };
+
+  sourceRoot = ".";
+
+  installPhase = ''
+    mkdir -p $out/share/icons
+    cp -r phinger-cursors $out/share/icons
+    cp -r phinger-cursors-light $out/share/icons
+  '';
```

Comment:
```suggestion
  installPhase = ''
    runHook preInstall
    
    mkdir -p $out/share/icons
    cp -r phinger-cursors $out/share/icons
    cp -r phinger-cursors-light $out/share/icons
    
    runHook postInstall
  '';
```

Reply:
Would be better to recursively copy all directories (variants) if possible instead of doing that manually. So future changes require less package maintenance, and phinger-cursors comes with a cursor theme build pipeline that allows to create new variants by just adding a new sprite in a figma document.